### PR TITLE
Wip/superm1/more fwupdate debugging

### PR DESCRIPTION
**Pull request by superm1**

### CODE REVIEW

File: plugins/uefi/fu-plugin-uefi.c [hunk 1]
```diff
@@ -572,6 +572,26 @@ fu_plugin_uefi_coldplug_resource (FuPlugin *plugin, fwup_resource *re)
...
+	gint rc;
+
+	rc = efi_get_variable (guid, "SecureBoot", &data, &data_size, &attributes);
+	if (rc < 0)
+		return;
+	if (data_size >= 1 && data[0] & 1)
+		result_str = "True";
+
+	g_debug ("SecureBoot is enabled: %s", result_str);
+	fu_plugin_add_report_metadata (plugin, "SecureBootEnabled", result_str);
```
Comment (hughsie) [inline plugins/uefi/fu-plugin-uefi.c]: Why not just `SecureBoot=enabled` and `SecureBoot=disabled`?

File: plugins/uefi/fu-plugin-uefi.c [hunk 2]
```diff
@@ -572,6 +572,26 @@ fu_plugin_uefi_coldplug_resource (FuPlugin *plugin, fwup_resource *re)
...
 }
 
+static void
+fu_plugin_uefi_test_secure_boot (FuPlugin *plugin)
+{
+	const efi_guid_t guid = EFI_GLOBAL_GUID;
+	const gchar *result_str = "False";
+	g_autofree guint8 *data = NULL;
+	gsize data_size = 0;
+	guint32 attributes;
```
Comment (hughsie) [inline plugins/uefi/fu-plugin-uefi.c]: Maybe also set to zero?

File: plugins/uefi/fu-plugin-uefi.c (hunk 1 above)
Comment (superm1) [reply plugins/uefi/fu-plugin-uefi.c]: Fixed

File: plugins/uefi/fu-plugin-uefi.c (hunk 2 above)
Comment (superm1) [reply plugins/uefi/fu-plugin-uefi.c]: Fixed

**Pull request merged**